saleor/graphql/product/schema.py

```diff
@@ -201,6 +201,18 @@ class ProductQueries(graphene.ObjectType):
         price_gte=graphene.Float(
             description="Filter by price greater than or equal to the given value."
         ),
+        minimal_price_lte=graphene.Float(
```

Comment:
We should now only use `filter` object to keep all available filters. These two fields that we add here are not necessary. `price_gte` and `price_lte` should also be removed but I'm not sure if it isn't used currently by our storefront.

Reply:
I've removed the `minimal_price_(lte|gte)` input fields and also added `DEPRECATION` warning in the `price_(lte|gte)` fields.